docs/src/examples/components/Button/SplitButton/SplitButtonExample.shorthand.tsx

```diff
@@ -0,0 +1,42 @@
+import * as React from 'react'
+import { SplitButton } from '@stardust-ui/react'
+
+const handleMainOptionClick = e => {
+  alert('PDF is all around us!')
+}
+const items = [
+  {
+    content: 'Export to PDF',
+    icon: 'files-pdf',
+    onClick: handleMainOptionClick,
+  },
+  {
+    content: 'Export to PDS',
+    icon: 'files-photoshop',
+    onClick: e => {
+      alert('Photoshop is awesone!')
+    },
+  },
+  {
+    content: 'Export as GIF',
+    icon: 'files-gif',
+    disabled: true,
+  },
+  {
+    content: 'Export to EPS',
+    icon: 'files-illustrator',
+    onClick: e => {
+      alert('Illustrator is kinda cool!')
+    },
+  },
+]
+
+const SplitButtonExampleShorthand = () => (
+  <SplitButton
+    onClick={handleMainOptionClick}
+    menuButton={{ menu: { items } }}
```

Comment:
Is there a case that a user would want to pass to `menuButton` prop something else than `items`?
Thinking from API point of view, having both `button` and `menuButton` prop can be confusing.
I would rather have `menu` or `items` instead of `menuButton`

Reply:
agree. we have `menu` in other components (MenuItem, MenuButton) - you can pass there the shorthand for the menu or just an array of items, the factories are smart enough to do the right thing

Reply:
addressed